Update oms-solution-update-management.md

shaikshafi77 opened the pull request:
The update Management is still in preview according to the portal. But it is not mentioned in the documentation. That is confusing the users and customers. So adding the preview in the title until it moves to GA

azurecla commented:
Hi __@sebastienros__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sebros). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Thraka commented:
@tamram Can you confirm the fix?

sebastienros commented:
To help you, in this code:

    CloudStorageAccount storageAccount = CloudStorageAccount.Parse(CloudConfigurationManager.GetSetting("StorageConnectionString"));

`CloudConfigurationManager` is actually in `Microsoft.WindowsAzure`.

shaikshafi77 committed:
Update oms-solution-update-management.md

The update Management is still in preview according to the portal. But it is not mentioned in the documentation. That is confusing the users and customers. So adding the preview in the title until it moves to GA
